index.rst

```diff
@@ -240,6 +240,101 @@ fitSrcMatchAstrometry
     :name: pl-jfb-ambitious-nofakes-01
 
 
+The Great Calibration Refactor Proposal: February 2023
+------------------------------------------------------
+
+Goals and non-goals
+"""""""""""""""""""
+
+We're replacing essentially everything between ISR and coaddition.
+ISR and coaddition will also see major changes (spurred by Calibpalooza and cell-based and chi-squared coadds, respectively) on similar timescales (we hope), but we're considering those out-of-scope and mostly orthogonal.
+While we can and should implement this piecemeal when we can, we want a complete vision of what it will look like in the end, and in some cases it may be easier to replace many tasks at once.
+
+We lean towards merging PipelineTasks with the same dimensions that run back-to-back rather than keeping them distinct.
+This is a bit of a shift - many smaller PipelineTasks leads to more flexibility via just pipeline definition changes, which has been very useful in prototyping, but we believe we are exiting the prototyping phase and should instead prioritize the I/O optimization and pipeline-simplicity advantages of having fewer bigger PipelineTasks.
+We very much intend to continue to delegate all real algorithmic work to subtasks; it's just that each PipelineTask will tend towards having more of those.
+
+This is probably our last best chance to get our naming conventions for dataset types and task labels under control, so we're including that in our proposal.
+
+Conventions
+"""""""""""
+
+- Task labels are camelCase and start with a lowercase verb: "associateIsolatedStars" instead of "isolatedStarAssociation".
+- Dataset type names are snake_case nouns preceded (if necessary) by adjectives: e.g. "initial_visit_summary"
+- Use "source" instead of "src" or "sources".
+- Avoid "catalog" or "cat" in dataset type names; use "source", "stars", "object", or "matches" instead when appropriate.
```

Comment:
I am split on this, I agree that if you are in the know, it does not add any additional information, but if you are not and just read a list of data set types, seeing `stars` does not really tell me anything. Is it a stack of postage stamps of isolated stars? Is it a data set type used in creation of synthetic stars? At least if there is a `table` or `catalog` somewhere in there it makes it a bit clearer what it is.